homeassistant/components/generic_hygrostat/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+  "domain": "generic_hygrostat",
+  "name": "Generic hygrostat",
+  "documentation": "https://www.home-assistant.io/integrations/generic_hygrostat",
+  "dependencies": [
+    "sensor",
+    "switch"
+  ],
```

Comment:
```suggestion
```

Why are core platforms listed as dependencies  ?

Reply:
Hi! This is make it initialize after those platforms since it is building on top of them. Probably not necessary, since individual entities could still be initialized after, which should not be a problem, but I've seen it with `generic_thermostat`: https://github.com/home-assistant/core/blob/dev/homeassistant/components/generic_thermostat/manifest.json
Should I remove it?

Reply:
Okay, removed.